app/templates/public/index.hbs

```diff
@@ -12,24 +12,23 @@
           <h3 class="ui header">{{t 'Tickets'}}</h3>
         </div>
         <div class="ui segment">
-          {{#if model.event.isTicketingEnabled}}
-            {{public/ticket-list eventCurrency=model.event.paymentCurrency
-                                 ticketUrl=model.event.ticketUrl
-                                 isTicketingEnabled=model.event.isTicketingEnabled
-                                 data=model.tickets
-                                 order=model.order
-                                 attendees=model.attendees
-                                 code=code
-                                 currentEventIdentifier=model.event.identifier
-                                 createNewUserViaEmail=(action 'createNewUserViaEmail')
-                                 loginExistingUser=(action 'loginExistingUser')
-                                 isLoginModalOpen=isLoginModalOpen
-                                 placeOrder=(action 'placeOrder')
-                                 isLoading=isLoading
-                                 userExists=userExists
-                                 errorMessage=errorMessage
-                                 save='save'}}
-          {{else}}
+          {{!-- {{#if model.event.isTicketingEnabled}} --}}
```

Comment:
@shreyanshdwivedi Why the following changes ?

Reply:
Ah! I misunderstood the code. Fixing this!!

Reply:
@kushthedude fixed. Please review